src/main/java/net/minecraftforge/common/Tags.java

```diff
@@ -276,7 +276,6 @@ private static void init(){}
         public static final IOptionalNamedTag<Item> SEEDS_MELON = tag("seeds/melon");
         public static final IOptionalNamedTag<Item> SEEDS_PUMPKIN = tag("seeds/pumpkin");
         public static final IOptionalNamedTag<Item> SEEDS_WHEAT = tag("seeds/wheat");
-        public static final IOptionalNamedTag<Item> SHEARS = tag("shears");
```

Comment:
This shouldn't be removed, as it is still useful for recipes. Also, binary breaks are bad mmkay.

Reply:
Would a tool action ingredient be useful? Be trivial to make an ingredient that matches if the stack has a tool action

Reply:
I don't want to add the concept of using tool actions in recipes.

Reply:
Okay, I revert the shears tag removement